x-pack/plugins/monitoring/server/lib/alerts/fetch_ccr_read_exceptions.ts

```diff
@@ -133,6 +133,11 @@ export async function fetchCCRReadExceptions(
 
   const response = await esClient.search(params);
   const stats: CCRReadExceptionsStats[] = [];
+
+  if (!response.aggregations) {
+    return stats;
+  }
+
   // @ts-expect-error declare aggegations type explicitly
   const { buckets: remoteClusterBuckets = [] } = response.aggregations?.remote_clusters;
```

Comment:
Since we have that new check above, do we still need to keep the question mark here `response.aggregations?`?

Reply:
Nope, will change that!